world/quests/test/task-demo.lisp

```diff
@@ -0,0 +1,185 @@
+(quest:interaction :name tent-check :interactable tent :dialogue "
+~ player
+| (There's no one inside, but the bedroll suggests it was recently occupied.)
+| (Does an android require a bedroll? I recharge standing up.)
+| ! eval (activate 'fi-tent)
+| ! eval (complete 'fi-check)
+")
+; these error
+; | ? (not 'fi-tent-talk = true)
+; | ? (not 'fi-tent-talk)
+; | ['fi-tent-talk | ! eval (activate 'fi-tent)]
+(quest:interaction :name fi-tent :interactable fi :dialogue "
+~ player
+| I think we've got company.
+~ fi
+| Harris said these caves were deserted.
+| [has-more-dialogue Did you find anything else? | Keep your eyes open.]
+")
+(quest:interaction :name sign-main-check :interactable sign-main :dialogue "
+~ player
+| (It's the same symbol that I have on my uniform.)
+| (Perhaps it's a warning...) 
+| ! eval (activate 'fi-sign-main)
+| ! eval (complete 'fi-check)")
+(quest:interaction :name fi-sign-main :interactable fi :dialogue "
+~ player
+| That sign over there has the same symbol on it as my uniform. I think it has something to do with androids.
+~ fi
+| Then we're on the right track.
+~ player
+| It also resembles an arrow, don't you think? Pointing up over the mountain?
+~ fi
+| I know you want to be a detective, but now I think you're seeing things.
+| Remind me to check your pattern recognition software when we get back.
+| [has-more-dialogue You got more? | You'd better get going.]")
+(quest:interaction :name sign-check :interactable sign :dialogue "
+~ player
+| (The words were scratched into the wood with a knife. I can barely read it. Upscaling resolution and enabling zoom...)
+| //READ ME - NO LITTER U FUCKS - BOTS PATROL HERE//
+| (Short and not so sweet - that's Harris.)
+| ! eval (activate 'fi-sign)
+| ! eval (complete 'fi-check)
+")
+(quest:interaction :name fi-sign :interactable fi :dialogue "
+~ player
+| There's a sign in the cave - looked and sounded like only Harris can. It warned of robots on patrol.
+~ fi
+| Rogues... They're not uncommon in caves like these. They're like you but not as strong - and thankfully not as smart.
+~ player
+| Was that a compliment?
+~ fi
+| A fact.
+| [has-more-dialogue Anything else? | Now get going.]")
+(quest:interaction :name landslide-check :interactable landslide :dialogue "
+~ player
+| (A landslide... That's a memory I could do without.)
+| (There's human blood on the rock. I can't follow them through there - I need to find another way around.)
+| ! eval (activate 'fi-landslide)
+| ! eval (complete 'fi-check)
+")
+; | ! eval (let (landslide true))
+(quest:interaction :name fi-landslide :interactable fi :dialogue "
+~ player
+| There's been a landslide in the cave.
+~ fi
+| That's not unusual in these parts. But given that everything else seems wrong about this place...
+~ player
+| I hope the android isn't buried under there.
+~ fi
+| Androids are tougher than they look.
+~ player
+| I don't look tough?
+~ fi
+| *Smiles* I didn't say that.
```

Comment:
Yeah I figured we'd probably want to handle something like this with a bespoke chat head emote, but it felt like a nice dramatic action to call out at the time

Could there also be times where we want to portray actions between characters that we can't animate? E.g. at the end I have the Stranger holding a chip aloft. It would allow us to visualise nice character details, while the in-game models are frozen, without incurring art budget. I suppose bringing in a little bit of prose narration for the player's imagination.

Asterisking them looks kind of ugly too I think, though I've seen other games do this in their subtitles, and I think players understand what it means

Reply:
I'll be frank, when I saw the emote I cringed pretty hard. I think it just comes across as amateurish, especially when we have the emote profile system already in place.

I understand that players get what it means, it's a common technique in text chats and so forth, especially in times before emojis. Still, I think adding a profile animation when necessary, just leaving it out, or expressing it in dialogue would ultimately be better.

It's especially jarring when it's a very seldom used thing, since you aren't prepared for dialogue suddenly being used for actions as well.